### PR TITLE
Add VPCZones

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
Add toronto region
Add few utility functions
Removed VPCRegion for Montreal since it does not have respective VPC region

### CODE REVIEW

**Review by mkumatag — CHANGES_REQUESTED** [review]
Need needs a revisit based on the data what is been published in the powervs doc here - https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server

and also please confirm the powervs zones to the colocated vpc zones mapping

File: region.go [hunk 1]
```diff
@@ -124,20 +132,33 @@ var Regions = map[string]Region{
...
-		SysTypes:    []string{}, // Missing
+	"tor": {
+		Description: "Toronto, Canada",
+		VPCRegion:   "ca-tor",
+		COSRegion:   "ca-tor",
+		Zones:       []string{"tor01"},
+		SysTypes:    []string{"s922", "e980"},
+		VPCZones:    []string{"ca-tor-1", "ca-tor-2", "ca-tor-3"},
+	},
+	"us-south": {
```
Comment (Karthik-K-N) [inline region.go:163]: How about moving this next to dal region block and adding a comment stating why we added it as separate entry

Comment (dharaneeshvrd) [reply region.go:163]: I have kept it in same order to maintain the alpha ascending order. Just added a comment.

**Comment by dharaneeshvrd** [comment]
@mkumatag Refactored now, region is not going to be used anywhere, just used as a plain string to map the respective values. 
Segregated us-south as separate since key should be prefix to easily retrieve mapping from powervs zone, us-east also treated like that.
Regarding VPC zones, I have added all 3 zones for all the regions since it's possible to create subnets in them.

**Review by mkumatag — APPROVED** [review]
I will let @Karthik-K-N to LGTM

**Review by Karthik-K-N — APPROVED** [review]
/lgtm

**Pull request merged**